sections/announcement-bar.liquid

```diff
@@ -169,6 +265,28 @@
       "unit": "s",
       "label": "t:sections.announcement-bar.settings.change_slides_speed.label",
       "default": 5
+    },
+    {
+      "type": "header",
+      "content": "t:sections.announcement-bar.settings.header__3.content",
+      "info": "t:sections.announcement-bar.settings.header__3.info"
+    },
+    {
+      "type": "checkbox",
+      "id": "enable_country_selector",
+      "default": true,
+      "label": "t:sections.announcement-bar.settings.enable_country_selector.label"
```

Comment:
I wouldn't mind reusing the strings from header here. Honestly with 3 places these are being used now, this all could have been made a single generic/global set of translations. Since we already have translations back though, this is fine for now. Especially if we end up tweaking where these settings live in the future.